youtube_dl/extractor/mujrozhlas.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    ExtractorError,
+    js_to_json,
+)
+
+
+class MujRozhlasIE(InfoExtractor):
+    IE_NAME = 'mujRozhlas'
+    IE_DESC = 'https://www.mujrozhlas.cz/'
+    _VALID_URL = r'https?://www\.mujrozhlas\.cz/(?:[^/]+/)*(?P<id>[^/?#&]+)'
+    _TESTS = [
+        {
+            'url': 'https://www.mujrozhlas.cz/vinohradska-12/zadne-dalsi-pusy-konec-prchala-spol-muze-znamenat-babisuv-odchod-z-politiky-tipuje',
+            'md5': '34ecaa47f64079a63d6b80498c280e9d',
+            'info_dict': {
+                'id': '0c54ba72-93dd-3a29-b567-910d3d8c71a8',
+                'ext': 'mp3',
+                'title': 'Žádné další pusy. Konec Prchala a spol. může znamenat Babišův odchod z politiky, tipuje politolog',
+                'description': 'md5:ec0610bdb1f591061dbd224d2dd9c19e',
+            },
+        },
+        {
+            'url': 'https://www.mujrozhlas.cz/kazki/princi-ta-zliy-drakon',
+            'md5': 'cbad6f68db6dc4d6d798d69b5d258aa5',
+            'info_dict': {
+                'id': 'ec5f53b2-3910-448e-8e7f-d6d1a19f4926',
+                'ext': 'm4a',
+                'title': 'Принці та злий дракон',
+                'description': 'md5:b21701e09c2b509c4451194af7ac271b',
+            },
+            'params': {
+                'format': 'hls-128',
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        player_data = self._search_regex(
+            r'\bvar dl = ({[^\n]+});',
```

Comment:
Relax the RE, and unless you set the `s` flag `[^\n]` is implied by `.`:
```suggestion
            r'\bvar\s+dl\s*=\s*({.+});',
```